app/controllers/settings/admission_webhook_controller.rb

```diff
@@ -0,0 +1,67 @@
+# Settings::AdmissionWebhook is responsibe to manage all the requests
+# related to the kubernetes auditing feature.
+class Settings::AdmissionWebhookController < SettingsController
+  def index
+    @form = AdmissionWebhookForm.new
+    set_instance_variables
+  end
+
+  def create
+    @form = AdmissionWebhookForm.new(admission_webhook_params)
+    @errors = Pillar.apply(
+      admission_webhook_pillars,
+      unprotected_pillars: admission_webhook_pillars.keys
+    )
+    if @form.valid? && @errors.empty?
+      redirect_to settings_admission_webhook_index_path,
+        notice: "Admission webhook settings successfully saved."
+    else
+      set_instance_variables
+      render action: :index, status: :unprocessable_entity
```

Comment:
what about possible `@errors`? we should show them in an alert, would be a waste to throw them away

Reply:
Fixed.